Tribler/Main/vwxGUI/list_header.py

```diff
@@ -629,12 +629,15 @@ def toggle_grid(event):
 
         self.grid_button.Bind(wx.EVT_LEFT_UP, toggle_grid)
 
+        # Martijn: Workaround for assertions on Linux when adding a wx.SearchCtrl widget to a wx.Sizer
+        search_container_panel = wx.Panel(panel)
+
         if sys.platform == 'darwin':
-            self.search = wx.TextCtrl(panel)
+            self.search = wx.TextCtrl(search_container_panel)
```

Comment:
oh, sorry somehow I thought you where undoing the OSX hack because SearchCtrl worked fine now.